### PR TITLE
add .portable to the yuzu bot

### PR DESCRIPTION
**Pull request by Majora8120**
ports the .portable command from the citra bot to the yuzu bot

### CODE REVIEW

File: src/responses/yuzu.json [hunk 1]
```diff
@@ -49,6 +49,7 @@
...
       "drivers": { "reply": "yuzu can use custom drivers for Android devices with an Adreno 6XX/7XX GPU (Snapdragon SoC). There are no custom drivers for Mediatek, Exynos, or any other SoCs currently.\n\nTo use custom drivers, head to #android-drivers, open a driver thread that corresponds with your phone's GPU, and open the latest link posted in the thread.\nIf you don't know your phone's GPU, search for your phone on <https://www.gsmarena.com/>.\nDownload the .zip file from the link, go to the GPU driver manager in the Settings tab and select the downloaded driver you'd like to install."},
+      "portable": {"reply": "Go to <https://yuzu-emu.org/downloads/> and scroll down to `Builds`. Then under **Builds**, click on your operating system's icon to the right of the latest build available to download the build.\nExtract it and put it wherever you wish. Inside the extracted `Yuzu` folder, create a folder called `user`. This Yuzu should now store all of its config, save files and such inside of this `user` folder.\nCheck to make sure that this `user` folder is in the same folder that contains `yuzu(.exe)`\nNow run the `yuzu` executable found inside of the `yuzu` folder."},
```
Comment (sakuramipha) [inline src/responses/yuzu.json:52]:
```suggestion
      "portable": {"reply": "Go to <https://yuzu-emu.org/downloads/> and scroll down to `Builds`. Then under **Builds**, click on your operating system's icon to the right of the latest build available to download the build.\nExtract it and put it wherever you wish. Inside the extracted `yuzu` folder, create a folder called `user`. This yuzu should now store all of its config, save files and such inside of this `user` folder.\nCheck to make sure that this `user` folder is in the same folder that contains `yuzu(.exe)`\nNow run the `yuzu` executable found inside of the `yuzu` folder."},
```

**Review by FearlessTobi — APPROVED** [review]
looks ok to me

**Comment by liushuyu** [comment]
Can you rebase this pull request?

**Comment by Majora8120** [comment]
I'm not good with github and I do not know how to rebase a pr. I'm just going to close this pr and make a new one

**Pull request closed without merging**